src/main/charts/bitbucket/templates/statefulset.yaml

```diff
@@ -76,15 +76,17 @@ spec:
             {{ if .Values.ingress.host }}
             - name: SERVER_PROXY_NAME
               value: {{ .Values.ingress.host | quote }}
+            - name: SERVER_PROXY_PORT
+              value: {{ include "bitbucket.ingressPort" . | quote }}
             - name: SETUP_BASEURL
               value: {{ include "bitbucket.baseUrl" . | quote }}
             {{ end }}
-            - name: SERVER_PROXY_PORT
-              value: {{ .Values.ingress.port | quote }}
```

Comment:
move `SERVER_PROXY_PORT` into the same `if` block as `SERVER_PROXY_NAME`, they need to go together